mm2src/mm2_main/tests/mm2_tests/mm2_tests_inner.rs

```diff
@@ -1075,7 +1075,7 @@ fn test_withdraw_and_send_hd() {
     const TX_HISTORY: bool = false;
     const PASSPHRASE: &str = "tank abandon bind salon remove wisdom net size aspect direct source fossil";
 
-    let coins = json!([rick_conf(), tbtc_segwit_conf()]);
+    let coins = json!([rick_conf(), tbtc_segwit_conf(), eth_testnet_conf()]);
```

Comment:
This doesn't have `eth_testnet_conf()` in dev, looks like a problem due to merging.